src/util.hpp

```diff
@@ -496,8 +496,7 @@ inline bool contains(const Container & container, const Value & value)
 
 
 #if 1
-# include <SDL_types.h>
-typedef Sint32 fixed_t;
+typedef int32_t fixed_t;
```

Comment:
Why is it called "fixed_t"? To me that implies it's to be used as fixed-point real-number storage; if that's not the case, then @aginor definitely has a good point.